[Gardening][iPadOS] `media/video-background-playback.html` progressed on iPadOS platform

iidmsa opened the pull request:
#### be512238059cf9b57dc3e26c12918984fde919f7
<pre>
[Gardening][iPadOS] `media/video-background-playback.html` progressed on iPadOS platform
<a href="https://bugs.webkit.org/show_bug.cgi?id=291025">https://bugs.webkit.org/show_bug.cgi?id=291025</a>
<a href="https://rdar.apple.com/problem/148550815">rdar://problem/148550815</a>

Unreviewed test gardening.

Removing test expectations.

* LayoutTests/platform/ipad/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/293545@main">https://commits.webkit.org/293545@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a54d6f952fdb31efdf78d16c3965696824509cd7

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/99081 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/18728 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/8974 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/104201 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/49662 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/19012 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/27160 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/75413 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/32533 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/102086 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/14449 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/89472 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/55777 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/14240 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/7444 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/49038 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/84171 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/7532 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/106564 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/26192 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/19084 "Found 1 new test failure: imported/w3c/web-platform-tests/IndexedDB/idbfactory-open-request-error.html (failure)") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/84371 "Found 1 new test failure: imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-type-on-non-root.html (failure)") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/26552 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/85672 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/83881 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/28549 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/6225 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/19903 "The change is no longer eligible for processing.") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/16141 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/26129 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/25952 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/29265 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/27518 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

bls1999 reviewed (commented):
From your PR branch, please run `Tools/Scripts/run-webkit-tests --ios-simulator --print-expectations media/video-background-playback.html` to verify this test will still run (expected should be `PASS`). Sometimes tests require explicit pass expectations as a directory has been skipped.

bls1999 reviewed (approved):
r+ but please resolve the merge conflict

bls1999 reviewed (changes requested):
Based on this output @iidmsa sent me, this should be an explicit PASS expectation to run on iPad:

```
% Tools/Scripts/run-webkit-tests --ios-simulator --print-expectations media/video-background-playback.html
Found 1 test; running 1, skipping 0.

Tests to run for iPhone 12 running iOS (1)
media/video-background-playback.html  ['PASS']

Tests to run for iPad (9th generation) running iOS (0)
```

bls1999 reviewed (approved):
LGTM! Thanks for making that edit. :)

iidmsa committed:
[Gardening][iPadOS] `media/video-background-playback.html` progressed on iPadOS platform
https://bugs.webkit.org/show_bug.cgi?id=291025
rdar://problem/148550815

Unreviewed test gardening.

Removing test expectations.

* LayoutTests/platform/ipad/TestExpectations:

Canonical link: https://commits.webkit.org/293545@main
